samples/04_gis_analysts_data_scientists/address-standardization-and-correction-with-SequenceToSequence.ipynb

```diff
@@ -0,0 +1,852 @@
+{
```

Comment:
<p>Please remove the credentials and replace it with <strong>gis = GIS('home')</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/870/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
update via https://github.com/Esri/arcgis-python-api/pull/870/commits/12ffaa2bd22c410c0ef092a47b2b5e1636713d2ca